### DISCUSSION TITLE
Build error with "custom.scss"

### DISCUSSION THREAD

**Discussion by evanwill** [CollectionBuilder/collectionbuilder.github.io#73] [Announcements]
If you are using a CB template copied before 2022-12-23, with the most up-to-date Jekyll packages, you might encounter a build error with the build failing on "custom.scss". The CB templates have been updated to fix this issue!

To fix it in your own existing template the best option is to rename the "custom.scss" file:

1. look for the "custom.scss" file in the folder "assets/css/", and rename the file to "cb.scss"
2. Edit the include "_includes/head/head.html", changing the line where it says `<link rel="stylesheet" href="{{ '/assets/css/custom.css' | relative_url }}" type="text/css">` to be `<link rel="stylesheet" href="{{ '/assets/css/cb.css' | relative_url }}" type="text/css">`

Alternatively, in your older existing projects, you can prevent Jekyll from updating the jekyll-sass-converter gem which is causing the issue. To do this, edit the "Gemfile" in your project, adding `gem "jekyll-sass-converter", "< 3.0"` on a new line. Delete your old "Gemfile.lock" then proceed as normal (i.e. `bundle exec jekyll s`). The older version of jekyll-sass-converter will work.

**Comment by dcnb** [comment]
The build error looked like this for one of our sites: 

Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/custom.scss':
                  expected "{".

**Reply by mldhistory** [reply]
Thank you Evan! I just got this error on a new collection I was trying to edit and build in GH desktop and this solution solved the problem.
Marcie
